README.md

```diff
@@ -97,7 +97,7 @@ The different decomposition strategies can be found in the
 In the example below, the circuit is decomposed using the Z-Y-Z decomposer.
 
 ```python
-from opensquirrel.decomposer.aba_decomposer import ZYZDecomposer
+from opensquirrel.passes.decomposer.aba_decomposer import ZYZDecomposer
```

Comment:
Sorry for the last minute comment:

Because of the way the `__init__.py` file in the in the `decomposer` directory is configured, you can import the various decomposers directly from the `decomposer` directory, _e.g._,
```suggestion
from opensquirrel.passes.decomposer import ZYZDecomposer
```

This should work for all decomposers and maybe nice to add this change to this PR still...

Reply:
@GuyPuts Do you mind if I make these changes and finalize the PR before this branch becomes too much delayed?

Reply:
Picking it up right now. This does not work for the general decomposer, by the way